src/OpenSage.Game.Tests/Data/Tga/TgaFileTests.cs

```diff
@@ -18,6 +19,11 @@ public void CanReadTgaTextures()
         {
             InstalledFilesTestData.ReadFiles(".tga", _output, entry =>
             {
+                switch (Path.GetFileName(entry.FilePath))
+                {
+                    case "map ang rhudaur.tga":
+                        return; // Corrupt, or unreferenced.
```

Comment:
Is it corrupt, or unreferenced?

Reply:
the question is how exactly do i detect that? since the ini files of bfme are not yet completely parseable

Reply:
Search in the ini files for that filename. That’s how I discovered the other ones were unreferenced.

(I assume you’ve got all the files extracted from .big files into a folder somewhere... otherwise this is all much harder.)